<launch>
	<arg name="sigverse_ros_bridge_port"        default="50001" />
	<arg name="ros_bridge_port"                 default="9090" />

	<node name="pr2_teleop_key" pkg="sigverse_pr2_teleop_key" type="pr2_teleop_key" output="screen" launch-prefix="gnome-terminal -t 'PR2 teleop key' -- ">
		<!--
		<remap from="/joint_states"                  to="/joint_states" />
		<remap from="/base_controller/command"       to="/base_controller/command" />
		<remap from="/head_traj_controller/command"  to="/head_traj_controller/command" />
		<remap from="/torso_controller/command"      to="/torso_controller/command" />
		<remap from="/l_arm_controller/command"      to="/l_arm_controller/command" />
		<remap from="/r_arm_controller/command"      to="/r_arm_controller/command" />
		<remap from="/l_gripper_controller/command"  to="/l_gripper_controller/command" />
		<remap from="/r_gripper_controller/command"  to="/r_gripper_controller/command" />
		-->
	</node>

	<group ns="sigverse_ros_bridge">
		<node name="sigverse_ros_bridge" pkg="sigverse_ros_bridge" type="sigverse_ros_bridge" args="$(arg sigverse_ros_bridge_port)"/>
	</group>
	
	<include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" > 
		<arg name="port" value="$(arg ros_bridge_port)"/>
	</include>
</launch>

